README.md

```diff
@@ -74,71 +84,29 @@ or [Cloud Manager](https://docs.cloudmanager.mongodb.com/tutorial/manage-program
 ### Configuring `mongocli`
 
 Run `mongocli config` to set up your credentials, 
-this is optional and you can use [env variables](#environment-variables) instead.
-
-If you're working with Ops Manager or Cloud Manager you need to define the service using `--service`
-
-For Ops Manager, `mongocli config --service ops-manager`.
+this is optional, you can use [env variables](https://docs.mongodb.com/mongocli/stable/configure/environment-variables/) instead.
 
-For Cloud Manager, `mongocli config --service cloud-manager`.  
+If you're working with Atlas Gov, Ops Manager or Cloud Manager you need to define the service using `--service`
 
-### Environment Variables
+For Atlas Gov, `mongocli config --service cloudgov`.
 
-You can use a combination of the next env variables to override your profile settings
+For Ops Manager, `mongocli config --service ops-manager`.
 
-- `MCLI_PUBLIC_API_KEY`
-- `MCLI_PRIVATE_API_KEY`
-- `MCLI_PROJECT_ID`
-- `MCLI_ORG_ID`
-- `MCLI_OPS_MANAGER_URL`
-- `MCLI_PROFILE`
-- `MCLI_OUTPUT`
-- `MCLI_MONGOSH_PATH`
+For Cloud Manager, `mongocli config --service cloud-manager`.
 
 ### Shell Completions
 
-If you install via [homebrew](#hombrew-on-macos) there's nothing else to do. 
-For other installations please refer to your preferred shell instructions.
-
-#### Bash
-
-```bash
-$ source <(mongocli completion bash)
-```
-
-To load completions for each session, execute once:
-
-Linux:
-```bash
-mongocli completion bash > /etc/bash_completion.d/mongocli
-```
-  
-macOS:
-```bash
-mongocli completion bash > /usr/local/etc/bash_completion.d/mongocli
-```
-
-#### Zsh
-
-```bash
-source <(mongocli completion zsh)
-``` 
-
-To load completions for each session, execute once:
-
-```bash
-mongocli completion zsh > "${fpath[1]}/_mongocli"
-```
+If you install via [homebrew](#hombrew-on-macos) there's nothing else to do.
```

Comment:
Do you mean installing via homebrew, users get completion by default and don't need to take extra steps? If so, maybe change to:
If you install via [homebrew](#hombrew-on-macos) no additional actions are needed.